src/Microsoft.AspNetCore.Identity/SignInManager.cs

```diff
@@ -582,7 +594,7 @@ private ClaimsIdentity CreateIdentity(TwoFactorAuthenticationInfo info)
         }
 
 
-        private async Task<SignInResult> SignInOrTwoFactorAsync(TUser user, bool isPersistent, string loginProvider = null)
+        private async Task<SignInResult> SignInOrTwoFactorAsync(TUser user, bool isPersistent, string loginProvider = null, bool bypassTwoFactor = false)
```

Comment:
Weird, I will double check the branch and see why the diff is messed up, perhaps I forgot to push a commit